### PR TITLE
Wazo 2178 sip headers

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: content/uc-doc/administration/interconnections/introduction.md [hunk 1]
```diff
@@ -55,6 +55,56 @@ a SIP provider from a NAT environment.
...
+            ...,
+            ["set_var", "PJSIP_HEADER(add,<HEADER NAME>)=<HEADER VALUE>"]
+        ],
+        ...
+    }
+
+
+#### Incoming calls
+
+Sometimes it is neccessary to match incoming SIP INVITE against a specific header to route the
```
Comment (sduthil) [inline content/uc-doc/administration/interconnections/introduction.md:117]:
```suggestion
Sometimes it is necessary to match incoming SIP INVITE against a specific header to route the
```

File: content/uc-doc/administration/interconnections/introduction.md [hunk 2]
```diff
@@ -55,6 +55,56 @@ a SIP provider from a NAT environment.
...
+call to the appropriate SIP endpoint.
+
+This is useful in a multi tenant situation where multiple tenants use the same provider.
+
+If you provider sends the `X-Dest-User: abc123` header when you receive a call you should add a
```
Comment (sduthil) [inline content/uc-doc/administration/interconnections/introduction.md:122]:
```suggestion
If your provider sends the `X-Dest-User: abc123` header when you receive a call you should add a
```

File: content/uc-doc/administration/interconnections/introduction.md [hunk 3]
```diff
@@ -55,6 +55,56 @@ a SIP provider from a NAT environment.
...
+        ...
+    }
+
+
+#### Incoming calls
+
+Sometimes it is neccessary to match incoming SIP INVITE against a specific header to route the
+call to the appropriate SIP endpoint.
+
+This is useful in a multi tenant situation where multiple tenants use the same provider.
```
Comment (sduthil) [inline content/uc-doc/administration/interconnections/introduction.md:120]:
```suggestion
This is useful in a multi tenant situation where multiple tenants share the same provider.
```

**Review by sduthil — APPROVED** [review]

**Review by sduthil — CHANGES_REQUESTED** [review]
see above comments

**Comment by pc-m** [comment]
> see above comments

Fixed and rebased thanks

**Review by sduthil — APPROVED** [review]

**Comment by fblackburn1** [comment]
@pc-m tag mergeit is missing

**Pull request merged**